README.md

```diff
@@ -10,20 +10,18 @@ This library provides all functions necessary to de- and encode GIF files.
 
 The high level interface consists of the two types
 [`Encoder`](https://docs.rs/gif/0.10.1/gif/struct.Encoder.html) and [`Decoder`](https://docs.rs/gif/0.10.1/gif/struct.Decoder.html).
-They as builders for the actual en- and decoders and can be used to set various
-options beforehand.
 
 ### Decoding GIF files
 
 ```rust
 // Open the file
 use std::fs::File;
-use gif::SetParameter;
-let mut decoder = gif::Decoder::new(File::open("tests/samples/sample_1.gif").unwrap());
+let input = File::open("tests/samples/sample_1.gif").unwrap();
 // Configure the decoder such that it will expand the image to RGBA.
-decoder.set(gif::ColorOutput::RGBA);
+let mut options = gif::DecodeOptions::new();
+options.set_color_output(gif::ColorOutput::RGBA);
 // Read the file header
-let mut decoder = decoder.read_info().unwrap();
+let mut decoder = options.read_info(input).unwrap();
```

Comment:
`read_info` isn't the most obvious method naming here, but I don't think it is a big deal

Reply:
Maybe. It was the previous name, we can always add an alias for `open`. Not a big deal in terms of interface.